http/src/commonMain/kotlin/com/mirego/trikot/http/requestPublisher/HttpRequestPublisher.kt

```diff
@@ -37,13 +37,15 @@ abstract class HttpRequestPublisher<T>(
 
                 executeRequest(cancellableManager, requestBuilder).subscribe(
                     cancellableManager,
-                    onNext = {
+                    onNext = { response ->
```

Comment:
Is it tested?

Reply:
Unfortunately, there are no unit tests in our trikot.http module 😞. I tried adding a new test class, but I could not get it to resolve trikot.foundation & trikot.streams dependencies locally 😩. 

You've got a good point here, so I created a new issue (#55) to address our lack of unit tests in this lib.